src/components/form/_form.scss

```diff
@@ -114,6 +114,7 @@
 
   .#{$prefix}--form-item {
     @include type-style('body-short-01');
+    @include carbon--font-family('sans');
```

Comment:
I thought `type-style('body-short-01')` contains `carbon--font-family('sans')` - Did you find otherwise?

Reply:
yes I believe that line was added before https://github.com/IBM/carbon-components/pull/1965 was merged and I've now taken it out

Reply:
yes I believe that line was added before https://github.com/IBM/carbon-components/pull/1965 was merged and I've now taken it out